iOS/Flows/Home/_New/new_ConversationViewController.swift

```diff
@@ -27,8 +27,6 @@ class new_ConversationViewController: FullScreenViewController,
     var onSelectedThread: ((ChannelId, MessageId) -> Void)?
 
     override var inputAccessoryView: UIView? {
-        // This is a hack to make the input hide during the presentation of the image picker.
-        self.messageInputAccessoryView.alpha = UIWindow.topMostController() == self ? 1.0 : 0.0
```

Comment:
Removing this hack because the keyboard wasn't showing up after displaying the action sheet. We can find a safer hack later if we're still having problems with the image picker.